searx/static/themes/simple/src/less/autocomplete.less

```diff
@@ -2,10 +2,13 @@
 
 .autocomplete {
   position: absolute;
+  width: 44.7rem;
+  margin-top: 2.4rem;
   max-height: 0;
   overflow-y: hidden;
   text-align: left;
-  .rounded-corners;
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
```

Comment:
FYI `-webkit-border...-radius` (28a597e43e) and ` -moz-border...-radius` (8935b1d) are no longer needed.

> I think it would be cleaner to add this as a mixin: ```.rounded-bottom-corners;``` and do this in `mixins.less`

Thanks for the hint, I will implement this today .. I still working on this PR .. coming soon ...